components/style/values/specified/length.rs

```diff
@@ -758,8 +875,8 @@ impl CalcLengthOrPercentage {
             match value {
                 SimplifiedValueNode::Percentage(p) =>
                     percentage = Some(percentage.unwrap_or(0.) + p),
-                SimplifiedValueNode::Length(NoCalcLength::Absolute(Au(au))) =>
-                    absolute = Some(absolute.unwrap_or(0) + au),
+                SimplifiedValueNode::Length(NoCalcLength::Absolute(length)) =>
+                    absolute = Some(absolute.unwrap_or(0) + Au::from(length).0),
```

Comment:
Done for the scope of the function, did you mean convert it also in the definition for `CalcLengthOrPercentage`?